man/fi_gni.7.md

```diff
@@ -254,10 +306,40 @@ but adds the following parameter:
 GNIX_FAB_RQ_NAMO_FAX (Fetch AND and XOR) and GNIX_FAB_RQ_NAMO_FAX_S
  (Fetch AND and XOR 32 bit).
 
+The `initialize_auth_key` function allows the user to initialize the user and 
+provider registration limits for an authorization key prior to its first use.
+
 #NOTES
 
 The GNI provider sets the domain attribute *cntr_cnt* to the the CQ limit divided by 2.
 
+The GNI provider can use a maximum of 4K memory registrations per *node* when using FI_MR_SCALABLE.
+Please consider this limitation when placing multiple processes on each node. 
+
+The GNI provider sets the default user registration limit to 192 when using FI_MR_SCALABLE,
+and sets the default provider registration limit to 64. These limits are directly associated
+with the authorization key in use when creating the registration. If no authorization key
+is used when creating the registration, the registration is automatically bound to the same
+authorization key as the domain to which the registration belongs. 
+
+When using FI_MR_SCALABLE, the provider may make registrations which consume some of the 
+registrations set aside for the provider. This impacts the performance of FI_LOCAL_MR, which 
+relies on provider-created registrations.
+
+All memory registrations are associated with an authorization key, whether it is the provider
+default key(keylen=0) or a user-acquired key (key!=NULL, keylen!=0). Each authorization
+key is associated with a unique GNI network key. A GNI network key can only accomodate a single
+memory mode, whether it is FI_MR_BASIC, or FI_MR_SCALABLE. If a user attempts to open multiple 
+domains using different memory modes with the same authorization key, the provider will return
+-FI_EINVAL. 
+
+When using FI_MR_SCALABLE, the user may request keys beginning at 0, and ending at the 
+user registration limit for a given authorization key.
```

Comment:
This is affected by the global limit of 4 VMDH table. Fixing this might require doing offset by local rank or something.